roles/server/defaults/main.yml

```diff
@@ -35,9 +35,15 @@ domain: '{{ inventory_hostname }}'
 hardware:
 - cpu: 'host'
 - vcpus: '2'
-- memory: '1024'
+
+- memory: '2048'
+
 - virt_type: 'kvm'
-- graphics: 'spice,listen=socket'
+
+# -ji- Todo: only use spice grafik for older qmu versions. will fail on my machine
+#- graphics: 'spice,listen=socket'
+- graphics: 'vnc,listen=0.0.0.0'
+
```

Comment:
here i changed the default graphics module, but we can actually check the kvm version before. i had libvirt already installed before i used your collection.... maybe thats the reason